conjure_oxide/tests/integration/minion_constraints/div_undefzero-02-zero/input.eprime

```diff
@@ -0,0 +1,10 @@
+language ESSENCE' 1.0
+
+find x : int(1..5)
+find z : int(1..2)
+
+$ x/0 =z is always false 
+
+$ although div_undefzero is true for (x/0 = z), we add our own undefined semantics.
+$ x / y = z ~> div_undefzero(x,y,z) /\ y != 0
+such that !(x / 0 = z)
```

Comment:
this seems to be rewritten to true (correctly) but due to this it's not a great test? ok to keep this one, but maybe add another where instead of x/0 you have x/y where y is find y : int(0..2). so it is potentially 0, but doesn't have to be. this way we should see a div_undefzero in the minion output (currently there isn't one)

Reply:
I do this in the sequel PR for this, but I'll change it here too